esti/golden/lakectl_identity_with_mail.golden

```diff
@@ -0,0 +1,3 @@
+User ID:          esti
```

Comment:
Do we need this file too?

Reply:
Not really, only in case we solve the esti email issue. We could add it when that happens